src/lib/appState.ts

```diff
@@ -4,6 +4,7 @@ import { ApiResponse } from "./apiResponses"
 export class AppState {
     compilationIsLoading: boolean = false
     apiResponse: ApiResponse = null
+    repeats?: number = 1
```

Comment:
Why does repeats need to be in the appState? would be nice if we could keep this dev feature to the compiler page

Reply:
Lattice View needs the repeats so it knows how large the lattice is, in order to create a working area that is the correct size. Maybe there is another way for LatticeView component to get that value

Reply:
@gwwatkin OK I removed repeats from appState. It's always a bit more cumbersome to implement prop drilling because you have to re-write the prop input types. But I got it done.